superset-frontend/src/SqlLab/components/ResultSet.tsx

```diff
@@ -476,6 +476,18 @@ export default class ResultSet extends React.PureComponent<
     return <div className="noControls" />;
   }
 
+  rowsReturned() {
+    return (
+      <div className="ReturnedRows">
+        {this.state.data && this.state.data.length >= 0 && (
+          <span>
+            {t(`%s rows returned`, this.props.query.results.data.length)}
```

Comment:
it feels like this length should be referencing the same length as above?

Reply:
Also, the query has a rows prop which may be easier than totaling.

Reply:
I originally had this as a drawing from the query function, but when I did that I got an error that said that Query has no property called rows, which is what is being passed into props.query. 

However, when I render with this.props.query.rows, it renders perfectly fine, and is responsive to how many lines the SQL result is. I changed it to this because I was tired of my terminal yelling at me.